### PR TITLE
[sp-stress] Ensure builds are reproducible. JB#57224

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

**Review by saukko — APPROVED** [review]

**Review by vigejolla — APPROVED** [review]
LGTM

File: src/cpuload.c [hunk 1]
```diff
@@ -297,7 +297,7 @@ int main(int argc, const char* argv[])
    const char *name;
    unsigned int load = 0;
 
-   printf ("\nCPU load generator, build %s %s.\n", __DATE__, __TIME__);
+   printf ("\nCPU load generator\n");
```
Comment (Thaodan) [inline src/cpuload.c:300]: You could replace __DATE__ and __TIME__with version which wouldn't change unless the package changes.

File: src/swpload.c [hunk 2]
```diff
@@ -453,7 +453,7 @@ int main(const int argc, const char* argv[])
   this_epoch = time(NULL);
   this_pid   = getpid();
   pagesize   = (unsigned)getpagesize();
-  printf ("stress paging/swapping load generator, build %s %s\n", __DATE__, __TIME__);
+  printf ("stress paging/swapping load generator\n");
```
Comment (Thaodan) [inline src/swpload.c:456]: same here

File: src/cpuload.c (hunk 1 above)
Comment (martyone) [reply src/cpuload.c:300]: Yeah but is it worth the money? We use RPM, so you rarely need to rely on build metadata embedded in the code. Hence I think it's better to simply drop such info.

**Pull request merged**